mappings/net/minecraft/client/render/BlockRenderer.mapping

```diff
@@ -19,14 +159,85 @@ CLASS net/minecraft/class_2336 net/minecraft/client/render/BlockRenderer
 		ARG 2 x
 		ARG 3 y
 		ARG 4 z
-	METHOD method_9384 (Lnet/minecraft/class_160;IF)V
+	METHOD method_9378 setBlockProperties (Lnet/minecraft/class_160;)V
+		ARG 1 block
+	METHOD method_9379 renderTorch (Lnet/minecraft/class_160;DDDDDI)V
+		ARG 1 block
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+		ARG 8 xOfset
+		ARG 10 yOfset
+		ARG 12 metadata
+	METHOD method_9380 renderBlockBottom (Lnet/minecraft/class_160;DDDLnet/minecraft/class_2616;)V
+		ARG 1 block
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+		ARG 8 sprite
+	METHOD method_9381 getTexture (Lnet/minecraft/class_160;I)Lnet/minecraft/class_2616;
+		ARG 1 block
+		ARG 2 dir
+	METHOD method_9382 renderPlantBlock (Lnet/minecraft/class_160;IDDD)V
+		ARG 1 block
+		ARG 2 metadata
+		ARG 3 x
+		ARG 5 y
+		ARG 7 z
+	METHOD method_9383 renderStemBlock (Lnet/minecraft/class_160;IDDDD)V
+		ARG 1 block
+		ARG 2 metadata
+		ARG 3 blockMaxY
+		ARG 5 x
+		ARG 7 y
+		ARG 9 z
+	METHOD method_9384 blockEntityRenderer (Lnet/minecraft/class_160;IF)V
 		ARG 1 block
+		ARG 2 metadata
 		ARG 3 tickDelta
-	METHOD method_9386 (Lnet/minecraft/class_160;III)V
+	METHOD method_9385 getTexture (Lnet/minecraft/class_160;II)Lnet/minecraft/class_2616;
+		ARG 1 block
+		ARG 2 dir
+		ARG 3 metadata
+	METHOD method_9386 renderMovingBlock (Lnet/minecraft/class_160;III)V
+		ARG 1 block
+		ARG 2 x
+		ARG 3 y
+		ARG 4 z
+	METHOD method_9387 blockRenderer2 (Lnet/minecraft/class_160;IIIFFF)Z
```

Comment:
Similar to the comment on the `blockEntityRenderer` method (perhaps this is `renderBlock`?)